### ISSUE TITLE
Wretched disappearing 

### ISSUE THREAD

**Issue by kadash91** [dalaranwow/dalaran-wow#4510]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

_______________________________________________________________________________________________________________

In Silvermoon ruins where you get Thealis's head, several of the Wretched continue to either disappear or just clip through the ground making it difficult to fight and loot them.

**Comment by Rushor** [comment]
Please apply ids from the creature or quest from http://wotlk.openwow.com

**Comment by Rushor** [comment]
okay i cannot find anything here https://www.google.de/#q=+Thealis%27s+head+wow+item&*

And btw, the issue where creatures drop below the texture/wall was fixed on PTR already.

Commit: 

Solve many of the creature falling through floor issues.

This seems entirely wrong to discount the returned map height if we're
already below it. In many cases this is exactly why we want the height!

* commit: bebb8db8808beafeab3d3f64cbc4eca5d4ca2ef1

**Comment by Rushor** [comment]
wrong label, it already works on PTR.

**Comment by TeridanDW** [comment]
These creatures http://wotlk.openwow.com/npc=15644 (teleport to creature 56373 and wait for few minutes) still fall under the textures when they use random movement.
![image](https://cloud.githubusercontent.com/assets/6797611/24496336/a1f3d81c-1540-11e7-8406-76bcbf9173cb.png)

**Comment by Rushor** [comment]
that was fixed, when i reworked the whole randommovement generator

Game/MovementGenerators: Randommovement - Enable Movement in deep water

* updates: https://github.com/dalaranwow/dalaran-wow/issues/4621
* commit: f368e376ef02c0541e16b5f7df9cadb7e98e1376

I will move this to "waiting for PTR Update" since the reworked randommovement mechanic is required to fix that behaviour here.

![GitHub Logo](http://i.imgur.com/MOFWYcT.jpg)
